cfgov/housing_counselor/jinja2/housing_counselor/improved_index.html

```diff
@@ -166,91 +166,94 @@ <h2 class="h4">
-                                            </p>
-                                            <p>
-                                                {% if counselor.weburl %}
-                                                <b>Website:</b>
-                                                <a class="a-link a-link__icon" href="{{ counselor.weburl }}">
-                                                    <span class="a-link_text">{{ counselor.weburl }}</span>
-                                                    {{ svg_icon( 'external-link' ) }}
-                                                </a>
-                                                <br>
-                                                {% endif %}
+                                        <ol start="{{ loop.index }}">
+                                            <li>
+                                                <p>
+                                                    {% if counselor.weburl %}
+                                                    <a class="a-link a-link__icon"
+                                                       href="{{ counselor.weburl }}">
+                                                      <span class="a-link_text">
+                                                          <strong>{{ counselor.nme }}</strong>
+                                                      </span>
+                                                      {{ svg_icon( 'external-link' ) }}
+                                                    </a>
+                                                    {% else %}
+                                                    <strong>{{ counselor.nme }}</strong>
+                                                    {% endif %}
 
-                                                {% if counselor.phone1 %}
-                                                <b>Phone:</b>
-                                                <a href="tel:+1-{{ counselor.phone1 }}">
-                                                    {{ counselor.phone1 }}
-                                                </a>
-                                                <br>
-                                                {% endif %}
+                                                    <br>
+                                                    {{ counselor.adr1 }}
+                                                    {% if counselor.adr2 and counselor.adr2 != ' '  %}
+                                                    <br>
+                                                    {{ counselor.adr2 }}
+                                                    {% endif %}
+                                                    <br>
+                                                    {{ counselor.city }}, {{ counselor.statecd }} {{ counselor.zipcd }}
+                                                </p>
+                                                <p>
```

Comment:
This particular `<p>` should be converted to a `<dl>`.

Reply:
I'll leave this to another PR since `<dl>` adds its own formatting from CF.

Reply:
[GHE]/CFGOV/platform/issues/3200

Thanks for the thorough review @Scotchester !